diagnostic.md

```diff
@@ -3,36 +3,40 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+Embers CLI, Ember usually will use much less code and look more elegant than most other
+frameworks, URL support
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+Templates are backed with context in both Handlebars and Ember, in Emeber the
+context is the controller
 ```
 
 Name at least two layers of an Ember application (1.0 or 2.0):
 
 ```sh
-# your answer here
+Routes and Controllers
 ```
 
 How does a component differ from an Ember template?
 
 ```sh
-# your answer here
+You can create your own tags with components
 ```
 
 What is an opinionated framework? Is Ember opinionated? How or how not?
 
 ```sh
-# your answer here
+Ember is an opinionated framework, people believe Ember is more elegant and
```

Comment:
Not quite what we were looking for. An opinionated framework favors convention over configuration. Ember is an opinionated
framework. It requires users to name files in certain ways, and use it's actions
and data structures in certain ways.
0/1